ui/app/templates/components/calendar-widget.hbs

```diff
@@ -34,18 +34,6 @@
             </div>
           </button>
         </li>
-        <li class="action">
```

Comment:
Again, removing all "Single Month" code that we had previously displayed in the calendar-widget dropdown.